### DISCUSSION TITLE
Add double tap to upvote

### DISCUSSION THREAD

**Discussion by DerasLTU** [Docile-Alligator/Infinity-For-Reddit#1205] [Ideas]
Add a double tap to upvote toggle in settings gestures menu. It would function similarly like it functions on the official Reddit app.

**Comment by DerasLTU** [comment]
Forgot to mention. It's only for comments.

**Comment by MysteriousOrb2000** [comment]
This is a needed feature since I accidentally up/down vote things in the comments.
